.pre-commit-config_template.yaml

```diff
@@ -57,6 +57,13 @@ repos:
     language: script
     pass_filenames: false
 
+  - id: delete-old-coverage-files
+    name: delete-old-coverage-files
+    description: Delete old coverage files.
+    entry: bash -c "find . -name ".coverage" -type f -delete"
```

Comment:
create a new file in `Tests/scripts` called `delete_old_coverage_files.sh` and call this script as we do in the above hook